features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -923,10 +923,17 @@
 <!--        <bundle>mvn:org.openhab.addons.bundles/org.openhab.extensionservice.marketplace.automation/${project.version}</bundle>-->
     </feature>
 
-    <feature name="openhab-misc-mqttbroker" description="Embedded MQTT Broker" version="${project.version}">
+    <feature name="openhab-misc-mqttbroker" description="MQTT Broker Moquette" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-mqtt</feature>
-        <bundle>mvn:org.openhab.io/org.openhab.io.mqttembeddedbroker/${project.version}</bundle>
+        <bundle dependency="true">wrap:mvn:io.netty/netty-common/4.1.34.Final</bundle>
+        <bundle dependency="true">wrap:mvn:io.netty/netty-buffer/4.1.34.Final</bundle>
+        <bundle dependency="true">wrap:mvn:io.netty/netty-transport/4.1.34.Final</bundle>
+        <bundle dependency="true">wrap:mvn:io.netty/netty-codec/4.1.34.Final</bundle>
+        <bundle dependency="true">wrap:mvn:io.netty/netty-resolver/4.1.34.Final</bundle>
+        <bundle dependency="true">wrap:mvn:io.netty/netty-handler/4.1.34.Final</bundle>
+        <bundle dependency="true">wrap:mvn:io.moquette/moquette-broker/0.12.1</bundle>
```

Comment:
After making a local distro build I found out that we need to add the following prerequisite to features in order to be able to use the wrap protocol in features:

`<feature prerequisite="true">wrap</feature>`

We might want to move that to the openhab-runtime-base feature so we don't have to duplicate it in many add-on features.

Also if the bundle name/symbolic name/version are not added to the wrap URL, these wrapped JARs show up very ugly with `bundle:list`:

```
220 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_api_api-common_1.7.0_api-common-1.7.0.jar
221 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_api_gax_1.42.0_gax-1.42.0.jar
222 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_auth_google-auth-library-credentials_0.13.0_google-auth-library-credentials-0.13.0.jar
223 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_auth_google-auth-library-oauth2-http_0.13.0_google-auth-library-oauth2-http-0.13.0.jar
224 │ Active │  80 │ 0                     │ wrap_file__home_wouter_software_openhab_instance_userdata_tmp_kar_openhab-addons-2.5.0-SNAPSHOT_com_google_http-client_google-http-client-jackson2_1.28.0_google-http-client-jackson2-1.28.0.jar
```

Can you add some parameters for these bundle manifest entries similar to #5219?

Reply:
Yes sure

Reply:
Very good! Also don't forget to add `<feature prerequisite="true">wrap</feature>` or the wrapping magic won't work. Or maybe you are preparing a PR to get it added to openhab-runtime-base?  :wink:

Reply:
> openhab-runtime-base?

Yup exactly

Reply:
Done: https://github.com/openhab/openhab-core/pull/676